extensions/wikibase/src/org/openrefine/wikibase/commands/FetchManifestCommand.java

```diff
@@ -0,0 +1,43 @@
+package org.openrefine.wikibase.commands;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.refine.commands.Command;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import static org.openrefine.wikibase.commands.CommandUtilities.respondError;
+
+import java.io.IOException;
+
+/**
+ * Proxies Wikibase manifests to allow the client to bypass CORS restrictions.
+ */
+public class FetchManifestCommand extends Command {
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+        throws ServletException, IOException {
+        String url = request.getParameter("url");
+        try {
+            if (url == null) {
+                respondError(response, "No URL provided.");
+                return;
+            }
+
+            // fetch the contents at the url with a plain get request and return the response
+            OkHttpClient client = new OkHttpClient();
+            Request req = new Request.Builder().url(url).build();
+            Response res = client.newCall(req).execute();
+            response.getWriter().write(res.body().string());
+            response.setContentType("application/json");
+            response.setStatus(200);
+            response.setCharacterEncoding("UTF-8");
```

Comment:
It would be worth cleaning up this part to just go through our generic utilities to return JSON from a command, which would probably involve parsing the JSON before (which should be fine?)
I think writing the response body before setting the encoding can cause encoding issues (if I remember correctly).